build.sbt

```diff
@@ -68,3 +69,13 @@ concurrentRestrictions in Global := Seq(
 autoAPIMappings := true
 
 coverageHighlighting := false
+
+// We only use sbt-release to update version numbers.
+releaseProcess := Seq[ReleaseStep](
+  inquireVersions,
+  setReleaseVersion,
+  commitReleaseVersion,
+  tagRelease,
+  setNextVersion,
+  commitNextVersion
+)
```

Comment:
I thought about this after we talked, and I'm wondering if it makes sense for us to make a single wiki page that covers the release of multiple packages. These scripts are similar by design so hopefully there will be a lot of overlap. Also in making these scripts I've had to scrape together information from multiple sources so it might be good to move it all to one place.